src/c_api.rs

```diff
@@ -49,6 +49,8 @@ pub struct GifskiSettings {
     pub once: bool,
```

Comment:
`Resize to max this width if non-0` => `Resize to max this width if non-0.` since you did that change in the other place already.